cli.py

```diff
@@ -258,3 +259,4 @@ def grade_problem(name, problem):
 
 if __name__ == "__main__":
     main()
+    print(f"{Style.RESET_ALL}")
```

Comment:
wrap the call to `main()` in a try-finally, so even if something goes wrong in `main`, the console styles aren't affected.